subjects/devops/better-cat/README.md

```diff
@@ -0,0 +1,64 @@
+## better-cat
+
+### Instructions
+
+Create a script `better-cat.sh` that will behave similarly to the cat command, but with additional functionality.
+
+The script should accept any number of filenames as arguments, and it should print the content of each file to the standard output. If no arguments are provided, the script should print the content of all files in the current directory.
+
+The script should also support the following options:
+
+- -c: If this option is provided, the script should exclude all lines that begin with the # character.
+- -l: If this option is provided, the script should print the line number before each line of output.
```

Comment:
Make the description of `-l` consistent with the solution implemented

Reply:
Done